index.js

```diff
@@ -1,65 +1,75 @@
-// ARRAY OF OBJECTS TO STORE A LIST OF BOOKS
+// BOOKS COLLECTION CLASS
 
-const bookCollection = [];
-
-// Add book function
-const form = document.getElementById('book-form');
-const { title, author } = form.elements;
-
-const addBook = function () {
-  form.addEventListener('submit', (e) => {
-    e.preventDefault();
-
-    const formData = {
-      title: title.value,
-      author: author.value,
-    };
-
-    bookCollection.push(formData);
-    localStorage.setItem('bookData', JSON.stringify(bookCollection));
-    window.location.reload();
-  });
-};
-
-// PRESERVE DATA IN THE BROWSER (LOCAL STORAGE)
-
-const fillForm = localStorage.getItem('bookData');
-
-if (fillForm) {
-  const data = JSON.parse(localStorage.getItem('bookData'));
-
-  bookCollection.push(...data);
+function Book(title, author) {
```

Comment:
- [ ] OPTIONAL: I recommend changing this "function" keyword to a class with a constructor.

Reply:
@Meltrust Thanks for the time. Will fix that soon.